writing-data

kghamilton89 opened the pull request:
Refactor and migration of [Writing Data](https://axibase.com/products/axibase-time-series-database/writing-data/). I added a brief blurb which links to this page under **Inserting Data** header on root `README.md` document.

kghamilton89 commented:
@rodionos For which page does this menu exist? I don't see it anywhere for files associated with this PR.

rodionos commented:
In Writing data on axibase.com we have many options like VMware, IBM. Didn't see them in the files as part of this PR.

https://axibase.com/products/axibase-time-series-database/writing-data/

![image](https://user-images.githubusercontent.com/2098022/44445031-2cf43e00-a5e8-11e8-98c4-09c878afc7b0.png)

kghamilton89 commented:
I've added the menu to the beginning of the document. I compressed the selections by group, so there are links to `/api`, `/collector/jobs`, `/integrations`, `parsers/csv`, and the VMware job. I just removed repeats or grouped like items, should I expand them or display them differently?
